InnerEye-DataQuality/InnerEyeDataQuality/deep_learning/self_supervised/simclr_module.py

```diff
@@ -2,7 +2,6 @@
 #  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think it would be good to attach ssl training run results for future reference - before and after this manual optimisation change.